### PR TITLE
Auto-create _matrix webhook in channel

### PR DESCRIPTION
**Pull request by anoadragon453**
Creates the _matrix webhook if it doesn't exist already.

### CODE REVIEW

File: src/bot.ts [hunk 1]
```diff
@@ -194,6 +194,10 @@ export class DiscordBot {
     if (botUser) {
       const webhooks = await chan.fetchWebhooks();
       hook = webhooks.filterArray((h) => h.name === "_matrix").pop();
+      // Create a new webhook if none already exists
+      if (!hook) {
+        hook = await chan.createWebhook("_matrix", "", "Matrix Bridge: Allow rich user messages");
```
Comment (Half-Shot) [inline src/bot.ts]: It would probably be good to wrap this in a try catch with some logging, since this would be a prime spot for permission failures. Gold star if you can check the bots permissions before trying to create the webhook also since that isn't guaranteed.

Comment (anoadragon453) [reply src/bot.ts]: Presumably we'll just get a permissions error if we don't have permission when creating the webhook. Would is thus be enough to just print out the error and move along or is it possible to request the permission somehow?

TL;DR do we need to check if trying to create it will tell us anyways.

**Comment by Half-Shot** [comment]
Yeah, this all looks fine. Merging.
